### PR TITLE
Create zh_tw.json

### PR DESCRIPTION
**Pull request by HJ-zhtw**
- Localized for Traditional Chinese (zh_tw)
- Original texts from 6.1.1 (en_us)
- Any suggestion/correction will be appreciated

### CODE REVIEW

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "options.showNone": "顯示未綁定",
   "options.availableKeys": "可用按鍵",
   "options.sort": "排序",
-  "options.category": "種類",
+  "options.category": "分類",
   "options.key": "按鍵",
   "options.toggleFree": "免切換",
```
Comment (HJ-zhtw) [inline src/main/resources/assets/controlling/lang/zh_tw.json]: Btw, I don't understand what "free" stands for.
I translated it as "no need to toggle" eventually.

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "options.search": "搜尋",
+  "options.showAll": "顯示全部",
```
Comment (Cutedobe) [inline src/main/resources/assets/controlling/lang/zh_tw.json]: 顯示全部

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "options.search": "搜尋",
```
Comment (Cutedobe) [inline src/main/resources/assets/controlling/lang/zh_tw.json]: 搜索

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  "options.showAll": "顯示全部",
+  "options.showConflicts": "顯示衝突",
```
Comment (Cutedobe) [inline src/main/resources/assets/controlling/lang/zh_tw.json]: 顯示沖突部分

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  "options.showNone": "顯示未綁定",
```
Comment (Cutedobe) [inline src/main/resources/assets/controlling/lang/zh_tw.json]: 顯示未綁定部分

File: src/main/resources/assets/controlling/lang/zh_tw.json (hunk 3 above)
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 根據 Google 和我自身經驗，臺灣人一般不講搜索。
搜索：4470 萬
搜尋：3 億 4100 萬

File: src/main/resources/assets/controlling/lang/zh_tw.json (hunk 4 above)
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 「沖突」非中華民國教育部所用詞，故採用「衝突」。
無加註「部分」一詞應不致理解困難，故暫不使用。

File: src/main/resources/assets/controlling/lang/zh_tw.json (hunk 5 above)
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 同上。

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  "options.availableKeys": "可用按鍵",
```
[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 此處原文非 Hotkey，故不使用快捷鍵。

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  "options.sort": "排序",
```
[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 不符合臺灣人用字習慣。

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+  "options.category": "分類",
+  "options.key": "按鍵",
```
[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 同上。

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  "options.toggleFree": "免切換",
```
[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 這一樣我的確不清楚，望能提供相關擷圖。：）

File: src/main/resources/assets/controlling/lang/zh_tw.json [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+  "options.confirmReset": "按一下以確定重設！"
```
[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 使用「按一下」或「點擊」皆不影響使用者理解。事實上，在原版 Minecraft 裡也有兩詞同時存在的情形。
Reset 一詞通常譯為「重設」，不應譯為「復位」。
文末驚嘆號應使用中文書寫使用的全形，而非英文的半形。

File: src/main/resources/assets/controlling/lang/zh_tw.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (Cutedobe) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: free可以指的是自由的，toggle可以指的是开关，也就是自由的开关，即自由切换

[the remark this answers is not in the document]
Comment (HJ-zhtw) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 詞意我知道喔，不清楚的是這功能的用途，因為我實際遊戲中找不到。

[the remark this answers is not in the document]
Comment (Cutedobe) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: 那你就联系一下作者吧

File: src/main/resources/assets/controlling/lang/zh_tw.json (hunk 1 above)
Comment (Cutedobe) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: https://www.mcbbs.net/thread-713187-1-1.html

Comment (Cutedobe) [reply src/main/resources/assets/controlling/lang/zh_tw.json]: options.toggleFree=显示未使用快捷键？

**Comment by jaredlll08** [comment]
Just let me know when you're done with the PR (I see a lot of conversation but I'm not exactly sure what is being said)

**Comment by Cutedobe** [comment]
> Just let me know when you're done with the PR (I see a lot of conversation but I'm not exactly sure what is being said)

Where is the function of "options. ToggleFree "in the game? What does toggle free stand for, so that the translation can work

**Comment by Cutedobe** [comment]
> > Just let me know when you're done with the PR (I see a lot of conversation but I'm not exactly sure what is being said)
> 
> Where is the function of "options. ToggleFree "in the game? What does toggle free stand for, so that the translation can work

Can be understood as Freedom to switch/switch freely/free switch

**Comment by jaredlll08** [comment]
So that actually isn't used in the mod right now, it used to be for a button that would show all keys (as in keyboard keys) that have not been assigned to a keybind.

Clicking the button would show this menu:
https://www.blamejared.com/content/zIcAX.webm

**Comment by Cutedobe** [comment]
> 因此，该功能现在实际上并未在mod中使用，它曾经是一个按钮，用于显示尚未分配给按键绑定的所有键（如键盘键）。
> 
> 单击该按钮将显示此菜单：[https](https://www.blamejared.com/content/zIcAX.webm) : 
> [//www.blamejared.com/content/zIcAX.webm](https://www.blamejared.com/content/zIcAX.webm)

Sorry, I may not understand

**Comment by jaredlll08** [comment]
The translation is never used.

**Pull request merged**